backend/src/lib.rs

```diff
@@ -244,6 +246,13 @@ impl LTN {
         self.neighbourhood =
             Some(Neighbourhood::new(&self.map, boundary.clone()).map_err(err_to_js)?);
 
+        // We can delete this assert if it's a valid code path, but I think it's not.
```

Comment:
Oh good spot, yeah. I forgot about this. I think you're right; this seems impossible now